docs/google/engine.md

```diff
@@ -0,0 +1,103 @@
+# Quantum Engine
+
+The Quantum Engine, via the `cirq.google.Engine` class, executes programs and jobs using the
+Quantum Engine API. 
+
+Note that the Quantum Engine API is not yet open for public access.
+
+## Authenticating to Google Cloud
+
+Before you begin, you will need to create a Google Cloud project with the API
+enabled and billing enabled.  You will then to create credentials in order to
+access the API.
+
+You can create application default credentials from the command line using the
+gcloud client:
+
+`gcloud auth application-default login`
+
+From a colab, you can execute:
+
+```
+from google.colab import auth
+auth.authenticate_user(clear_output=False)
+```
+
+More information on creating application default credentials can be found on the
+[Google cloud](https://cloud.google.com/docs/authentication/production) website.
+
+## Engine class
+
+The `Engine` class is the entry point to communicate with the API.
+
+It can be initialized using your project id (found within your
+[Google Cloud Platform Console](https://console.cloud.google.com)).
+You can use this instance to run quantum circuits or sweeps (parameterized
+variants of a general circuit).
+
+```
+import cirq
+
+# A simple sample circuit
+qubit = cirq.GridQubit(5, 2)
+circuit = cirq.Circuit(
+    cirq.X(qubit)**0.5,                 # Square root of NOT.
+    cirq.measure(qubit, key='result')   # Measurement.
+)
+
+# Create an Engine object to use.
+# Replace the project id with the id from your cloud project.
+engine = cirq.google.Engine(project_id='your_project_id',
```

Comment:
instead of a string, use the variable YOUR_PROJECT_ID. This will cause python to barf if they forget to replace it, instead of trying to make a connection.

Reply:
Done.